pom.xml

```diff
@@ -104,7 +107,34 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>jackson2-api</artifactId>
-            <version>2.8.11.3</version>
+            <version>2.11.1</version>
+        </dependency>
+        <dependency>
+            <groupId>io.jenkins.plugins</groupId>
+            <artifactId>checks-api</artifactId>
+            <version>0.1.0-rc4.fc43ef9447c7</version>
```

Comment:
release this?

Reply:
I'm ready to release it now, but we may need to refactor some part since we have to publish checks for  `Queue.WaitingItem` instead of `Run` for `queued` state, is that ok? (I believe nothing changes in the consumer side after the that)